cpp/tests/io/parquet_test.cpp

```diff
@@ -52,6 +52,9 @@
 #include <random>
 #include <type_traits>
 
+// Set this to true to enable printing, for debugging.
+auto constexpr print_enabled = false;
```

Comment:
Should this be read from env variable?

Reply:
I copied the mechanism from structs_tests. Either way, this is removed now :)